br/pkg/storage/parse_test.go

```diff
@@ -63,7 +63,6 @@ func TestCreateStorage(t *testing.T) {
 	require.Equal(t, "bucket2", s3.Bucket)
 	require.Equal(t, "prefix", s3.Prefix)
 	require.Equal(t, "https://s3.example.com", s3.Endpoint)
-	require.Equal(t, ks3SDKProvider, s3.Provider)
```

Comment:
add a UT for endpoint-based check

Reply:
`s3_test.go:TestS3URI` already tests it.